Demo/EarlGreyExample/Podfile

```diff
@@ -30,6 +30,9 @@ end
 target "EarlGreyExampleSwiftTests" do
   project 'EarlGreyExample'
 
+  # Make sure it is compatible with other pods, i.e. R.swift
+  pod 'R.swift'
```

Comment:
Do we have any R.swift code in this target? We use EGExample / EGExampleSwift for Sanity Testing our new pods. Might not want to add a new dependency on it.